platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/transaction/system/SystemTransactionExtractionUtils.java

```diff
@@ -65,21 +65,19 @@ private SystemTransactionExtractionUtils() {}
      */
     @SuppressWarnings("unchecked")
     public static @Nullable <T> List<ScopedSystemTransaction<T>> extractFromEvent(
-            @NonNull final BaseEvent event, @NonNull final Class<T> systemTransactionTypeClass) {
+            @NonNull final BaseEventHashedData event, @NonNull final Class<T> systemTransactionTypeClass) {
```

Comment:
this is going in the wrong direction. `BaseEventHashedData` should be deleted when the migration is done, so we should not have anything depend on this class

Reply:
Changing to `GossipEvent` instead